src/zenml/materializers/cloudpickle_materializer.py

```diff
@@ -29,7 +29,7 @@
 
 logger = get_logger(__name__)
 
-DEFAULT_FILENAME = "artifact.pkl"
+DEFAULT_FILENAME = "model.pkl"
```

Comment:
why? This could be anything, in most cases it's actually not a model.

Reply:
this was just testing but we need to fix it because vertex model registry can't detect sklearn model unless it's actually named `model` otherwise a code should be added to load it manually

Reply:
FYI this will break all existing pipelines that have used this materializer in the past. If you do need to keep this change, you must include some code that checks for the old file name, to preserve backwards compatibility.

Reply:
I have implemented new solution for this which follows what we talked about this morning however it doesn't cover already trained models